### PR TITLE
create multi clusters with same clustername in different base domain

### PR DESCRIPTION
**Pull request by jinyunma**
auto case: OCP-24432 [ipi-on-azure] Create multiple clusters with the same cluster name + the same region + different base domain

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-multi-clusters-diff-basedomain-arm-f28

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/cases/multi-clusters/same-clustername-diff-basedomain/deprovision/cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ref:
+  as: cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision
+  from: upi-installer
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/cases/multi-clusters/same-clustername-diff-basedomain/deprovision/cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision-ref.yaml:3]: I guess we can use `installer` image here.

Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/cases/multi-clusters/same-clustername-diff-basedomain/deprovision/cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision-ref.yaml]: yes, updated, also updated cucushift-installer-rehearse-azure-cases-multi-clusters-same-subnets-deprovision-ref.yaml together.

**Comment by jinyunma** [comment]
tested job: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58355/rehearse-58355-periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-multi-clusters-diff-basedomain-arm-f28/1851898283178332160

/pj-rehearse ack

@jianlinliu PTAL again, thanks.

**Pull request merged**